docs/ADVANCED.md

```diff
@@ -266,42 +304,70 @@ splunk:
   ...
 ```
 
-For more instructions on how to bring your own certificates, please see: https://docs.splunk.com/Documentation/Splunk/latest/Security/ConfigureSplunkforwardingtousesignedcertificates
+Fore further instructions, see [Configure Splunk forwarding to use your own certificates](https://docs.splunk.com/Documentation/Splunk/latest/Security/ConfigureSplunkforwardingtousesignedcertificates).
 
 ## Build from source
```

Comment:
I'm wondering if we should move this to `CONTRIBUTING.md`? I think we might want to tailor this `ADVANCED.md` section to outline/detail individual features of the Docker image and how it can be used by the end user. 

Building images is more of a developer workflow, and the majority of end users should not need to know how to build everything from scratch (hopefully, assuming we've done our jobs in delivering a usable image :P)

Reply:
The `docs/refactor` branch name is a bit of a misnomer here. I really only edited for content and clarity rather than the architecture of the docs, which is the next step. A bunch of sections definitely need to be shuffled around.

Reply:
I definitely agree that it needs to be separated out based on users and use cases.